src/scenes/Exports/styles.scss

```diff
@@ -1,11 +1,13 @@
+@import '@entur/tokens/dist/styles.scss';
 @import '@entur/button/dist/styles.css';
 
 .exports {
   flex: 1;
   display: flex;
   flex-direction: column;
-}
 
-.create-export-button {
-  width: 20rem;
+  .create {
```

Comment:
Unrelated change?

Reply:
Oh I see this is because of https://github.com/entur/enki/pull/1391/files#r1627427420